.github/workflows/build-to-dockerhub.yml

```
@@ -0,0 +1,45 @@
+name: Build Images and push them to Dockerhub
+on:
+  release:
+    types:
+      - published
+jobs:
+  release-image:
+    name: Release images
+    strategy:
+      matrix:
+        target:
+          - karmada-aggregated-apiserver
+          - karmada-controller-manager 
+          - karmada-scheduler
+          - karmada-webhook
+          - karmada-agent
+          - karmada-scheduler-estimator
+          - karmada-interpreter-webhook-example
+    runs-on: ubuntu-18.04
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@v1
+      - name: Set up Buildx
+        uses: docker/setup-buildx-action@v1
+      - name: Login to DockerHub
+        uses: docker/login-action@v1 
+        with:
+          username: ${{ secrets.REPO_USERNAME }}
+          password: ${{ secrets.REPO_PASSWORD }}        
```

```suggestion
          username: ${{ secrets.DOCKERHUB_REGISTRY_USER_NAME}}
          password: ${{ secrets.DOCKERHUB_REGISTRY_PASSWORD}}        
```
The secrects are added to the repo now.